app/models/state_file_nj_intake.rb

```diff
@@ -213,7 +213,7 @@ def medical_expenses_threshold
 
   def validate_state_specific_w2_requirements(w2)
     super
-    if w2.wages.positive? && !w2.state_wages_amount.positive?
+    if w2.wages.positive? && (w2.state_wages_amount.nil? || w2.state_wages_amount <= 0)
```

Comment:
[dust] I was thinking about the switch from `!positive` to `<= 0`. I think it's less Ruby-ish but more clear to read conditionally - I'm in favor of this change